docs/releases.md

```diff
@@ -30,6 +30,17 @@
 
 ## Master branch
 
+Lychee currently undergoes a major architectural refactoring of its internals.
+The master branch already includes the first part of that refactoring which mostly concentrates on the backend.
+
+These changes and information only affect those users who directly follow the master branch:
+
+- Despite the substantial changes your photos and albums should be as safe as with any other commit to the master branch. Don't worry.
```

Comment:
Honestly, I would advocate for more caution, such as recommending that users back up their DB before migrating. How about something like:

_While we have made every effort to ensure that your existing photos and albums are as safe as with any other commit to the master branch, due to the sheer size of current changes we recommend that you take additional precautions, such as creating a backup of your SQL database before migrating._

Reply:
This seems sensible. It's also worth remembering we've assured users that master is stable in the past.